src/test/versionbits_tests.cpp

```diff
@@ -1,456 +0,0 @@
-// Copyright (c) 2014-2021 The Bitcoin Core developers
```

Comment:
It might be useful to have a hereticactivation_tests unit test.

Reply:
Patches welcome :) (The versionbits_tests.cpp didn't seem like it could be reused very much, and the fuzz test seems like it does the job well enough)